app/jobs/bill_subscription_job.rb

```diff
@@ -5,10 +5,11 @@ class BillSubscriptionJob < ApplicationJob
 
   retry_on Sequenced::SequenceError
 
-  def perform(subscriptions, timestamp)
-    result = Invoices::CreateService.new(
+  def perform(subscriptions, timestamp, invoice_source: :initial)
+    result = Invoices::SubscriptionService.new(
       subscriptions: subscriptions,
       timestamp: timestamp,
+      invoice_source: invoice_source,
```

Comment:
Is this field there only for debugging purposes? I haven't seen that it's used actually...

Reply:
It's used only once [here](https://github.com/getlago/lago-api/pull/662/files#diff-c2fc569d4dc2c01c9cf24a63e8b53ba1b815903360e0fbdda0323fe4d9ada80cR170) and that's the key to make sure the subscription is not billed twice for the same period without impacting `terminate` invoices